CHANGES.rst

```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+2.0.6 (2016-05-1)
+------------------
+
+Fixes:
+
+-Changed the color of the navigation in the calendar widget to grey(from blue) using inherit.
```

Comment:
Also: Please add your changelog entry to the 2.0.9 section, which is still unreleased.